### DISCUSSION TITLE
Removing Old GitHub Account from Contributor List

### DISCUSSION THREAD

**Discussion by RazRosh** [community/community#47208] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I added a friend as a contributor to my GitHub repository but accidentally used their old GitHub account.
To fix this, I created an orphan branch, removed the old account from the repository, added their new account as a contributor, and merged the orphan branch back to the main branch.
However, the old account is still listed as a contributor, even though they have no commits or contributions. Is there a way to remove them from the contributor list, or is this a bug in GitHub?

[My Repo](https://github.com/RazRosh/Fluentd-Kafka)

![Screenshot 2023-02-13 075212](https://user-images.githubusercontent.com/47662109/218381139-96c35438-bfd8-48ea-8d7d-b058cea35068.png)

**Comment by jge162** [comment]
Hello @RazRosh it is possible that this is a caching issue on GitHub's end. Try clearing your browser's cache and cookies and reloading the page.

**Reply by RazRosh** [reply]
Hey, thanks for the reply!
I've tried it but it didn't work. So I do believe the problem is on GitHub's end.

**Reply by jge162** [reply]
yup most likely a caching issue, just gotta wait it out.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @RazRosh ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
